tempto-core/src/main/java/com/teradata/tempto/internal/hadoop/hdfs/HdfsModuleProvider.java

```diff
@@ -47,6 +55,15 @@ public Module getModule(Configuration configuration)
             @Override
             protected void configure()
             {
+                Set<String> configurationKeys = configuration.listKeys();
+                if (!configurationKeys.contains(CONF_HDFS_WEBHDFS_HOST_KEY)
+                        || !configurationKeys.contains(CONF_TESTS_HDFS_PATH_KEY)) {
+                    logger.debug("No HDFS support enabled as '{}' or '{}' is configured",
```

Comment:
I used had that, but then that line was printed multiple times (3 times during startup and about 2 times per test).